Rakefile

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require 'logger'
+require 'bundler'
+
+require 'yaml'
+
+Bundler.require
+Dotenv.load
```

Comment:
Sounds like we need an `environment.rb` to re-use in `runner.rb` and here instead of duplicating the minimal required setup.